pkg/reconciler/revision/resources/deploy.go

```diff
@@ -119,8 +119,8 @@ func rewriteUserProbe(p *corev1.Probe, userPort int) {
 	}
 }
 
-func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig *tracingconfig.Config, observabilityConfig *metrics.ObservabilityConfig, autoscalerConfig *autoscalerconfig.Config, deploymentConfig *deployment.Config) (*corev1.PodSpec, error) {
-	queueContainer, err := makeQueueContainer(rev, loggingConfig, tracingConfig, observabilityConfig, autoscalerConfig, deploymentConfig)
+func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig *tracingconfig.Config, networkConfig *network.Config, observabilityConfig *metrics.ObservabilityConfig, autoscalerConfig *autoscalerconfig.Config, deploymentConfig *deployment.Config) (*corev1.PodSpec, error) {
```

Comment:
I think Revision should not consume networkConfig as Revision should not have networking related knowledge.

IMO, we probably want to add a separate field within [observability configmap](https://github.com/knative/serving/blob/master/config/core/configmaps/observability.yaml#L15) to turn on/off the feature of exporting tag header based routing metrics instead of using the feature flag defined in networkConfig  to avoid breaking the Route/Revision abstraction. WDYT?

/cc @yanweiguo

Reply:
It is OK. 
(In fact, I also thought about the idea, but gave up because dependency between this PR and the tag based routing feature.)

BTW, as @tcnghia mentioned in https://github.com/knative/serving/pull/6036 , if `config-features` can be available soon, how about using it?

Reply:
Like https://github.com/knative/serving/pull/6036, I'll use `config-observability` at this time.

Reply:
@ZhiminXiang  I've realized that use of `config-observability` is little bit messy, because it requires change of knative/pkg as well. So, I would like to use `config-features`. 
How do you think about it?

Reply:
Since the `config-features` seems not available right now, I would like to go with `config-observability` at this time. After `config-features` is settled,  it would better to move this one there like #6036 .